.travis.yml

```diff
@@ -1,30 +1,16 @@
-# Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
 os:
-  - linux
   - osx
+  - linux
 julia:
-  - "0.6"
-  - "1.0"
-  - "nightly"
+  - 0.7
+  - 1.0
```

Comment:
A small thing: If you put the `"1.0"` into quotes, then Travis output will look nicer – Travis will show `1.0` instead of `1`.